src/http/server_context.rs

```diff
@@ -12,9 +13,16 @@ pub struct ServerContext {
   pub schema: dynamic::Schema,
   pub http_client: Arc<dyn HttpClient>,
   pub blueprint: Blueprint,
+  pub http_data_loaders: Arc<Vec<DataLoader<HttpDataLoader>>>,
+  pub gql_data_loaders: Arc<Vec<DataLoader<GraphqlDataLoader>>>,
```

Comment:
Do we need `Arc<Vec<_>>`? or can we use `Vec<Arc<_>>`? The second one might have lesser contention.